internal/backends/gl/Cargo.toml

```diff
@@ -20,6 +20,10 @@ path = "lib.rs"
 svg = ["resvg", "usvg", "tiny-skia"]
 wayland = ["winit/wayland", "glutin/wayland", "copypasta/wayland"]
 x11 = ["winit/x11", "glutin/x11", "copypasta/x11"]
+# dlopen fontconfig at runtime rather than link at build time.
+# useful for cross compiling
+# Using a vendored fontconfig C library does not work well, refer to Issue #88.
+fontconfig-dlopen = [ "yeslogic-fontconfig-sys/dlopen" ]
```

Comment:
IMHO we could just enable this by default, not sure we need it to be a feature at the moment.

Reply:
I discussed this with Fedora packagers recently with regard to [using dlib for the JACK Rust bindings](https://github.com/RustAudio/rust-jack/pull/165). They said they would prefer Rust packages to link C libraries. So I think it should be a feature. I lean towards leaving it off by default to ensure Slint applications comply with LInux packaging guidelines, but I don't have a strong opinion about whether the default is dlopen or linking.

Reply:
I think we can just let it the default for now.
If this turns out to be a problem, we can add features later.

Ideally, could it be the other way in yeslogic-fontconfig-sys?  dlopen the default, and a feature to do the linking? 
So downstream users can enable direct linking by adding a feature without having to propagate the feature through the whole stack.

Another option than a feature would be to add the ability to configure/override that with an environment variable read by the build script of the -sys crate.

Reply:
I like the idea to use an environment variable. build.rs could read an environment variable to toggle the Cargo feature via printing [cargo:rustc-cfg=feature="dlopen"](https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorustc-cfgkeyvalue). I agree that's a nicer solution than propagating the Cargo feature selection all the way down the stack of Cargo.toml's.

Reply:
Excellent point, I think the majority of users would probably want dlopen and the packaging might be the exception here. So opting into linkage might be easier.

Regarding packaging, I hope that if Slint gets packaged for a Linux distro it would be with the Qt backend instead of the GL backend.

Reply:
I think the fontconfig bindings should default to doing the right thing for packaging (linking, not dlopen). In practice I don't think it's a big issue to link at build time. If a developer of a Slint application has the build fail, the error message says why (pkgconfig couldn't find fontconfig), so it's not so hard to resolve. The confusion in #88 arose because servo-fontconfig silent did the wrong thing (building a vendored C library) instead of failing the build when the fontconfig-dev package wasn't installed.

I don't expect packagers to read the documentation of all the dependencies of an application they're packaging. I also don't expect application developers to proactively communicate to packagers about a need to set an environment variable for a dependency. This PR solves a problem for cross compiling from Linux to Linux on a different CPU architecture. I think it's fine to require setting an environment variable for this case.

Reply:
I implemented setting the `RUST_FONTCONFIG_DLOPEN=on` environment variable and removed exposing the Cargo feature down Slint's stack of Cargo.toml's.